cdap-ui/app/cdap/components/PipelineDetails/store/ActionCreator.js

```diff
@@ -18,13 +18,26 @@ import {MyPipelineApi} from 'api/pipeline';
 import {MyProgramApi} from 'api/program';
 import PipelineDetailStore, {ACTIONS} from 'components/PipelineDetails/store';
 
+const publish = (params, data) => {
```

Comment:
I am seeing functions like this but do we really need them? It looks like a one line function which just returns the api observable. Can't we do this in a place where it is used? I am missing to see the benefit here.

Reply:
I guess not really, I only added them so that components can just import the ActionCreator instead of importing `MyPipelineApi` or `MyProgramApi`